### ISSUE TITLE
Remove the reference to WCAG 1.2.3 in the help page for "video-caption" rule

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#2879]
Expectation: The reference to ```WCAG 1.2.3``` in [```video-caption``` rule's help page](https://dequeuniversity.com/rules/axe/4.1/video-caption) should be removed as per the change in PR https://github.com/dequelabs/axe-core/pull/832

**Comment by WilcoFiers** [comment]
Good catch, thank you! I'll fix that.

**Comment by padmavemulapati** [comment]
Verified with the latest code,
Not seeing `WCAG: 1.2.3` reference removed from the `video-caption` rule help pages
![image](https://user-images.githubusercontent.com/47409579/125153930-bc81a100-e174-11eb-97f8-72c315a93037.png)
